src/openvic-simulation/map/ProvinceInstance.hpp

```diff
@@ -135,6 +140,9 @@ namespace OpenVic {
 		bool setup(BuildingTypeManager const& building_type_manager);
 		bool apply_history_to_province(ProvinceHistoryEntry const& entry, CountryInstanceManager& country_manager);
 
+		void initialise_for_new_game();
+
 		void setup_pop_test_values(IssueManager const& issue_manager);
+		std::vector<Pop*> get_mutable_pops();
```

Comment:
There's no need to create a temporary vector of pops, you can just add a mutable getter like so:
```
plf::colony<Pop>& ProvinceInstance::get_mutable_pops() {
	return pops;
}
```
and then iterate through it normally:
```
for (Pop& pop : province.get_mutable_pops()) {
	// ...
}
```

Reply:
Great, it also saves me the null checks :)